### PR TITLE
feat(snapshot): add better logs to monitor command

### PR DESCRIPTION
**Pull request by y-lakhdar**
TODO: Merge #397 first.

## Proposed changes

Success monitor log
image

Error monitor log:
image

 

## Testing

- [x] Unit Tests:

- [ ] Functionnal Tests:

- [x] Manual Tests:

### CODE REVIEW

**Comment by louis-bompart** [comment]
@y-lakhdar if you want early review (i.e. before #397 is merged, target the branch of #397 ;) )

**Comment by y-lakhdar** [comment]
> @y-lakhdar if you want early review (i.e. before #397 is merged, target the branch of #397 ;) )

No, I messed up the merge so it contains changes from master and #397.
I'll clean it up once #397 is merged.

File: packages/cli/src/commands/org/config/monitor.ts [hunk 1]
```diff
@@ -63,14 +66,38 @@ export default class Monitor extends Command {
...
+    }
+  }
+
+  // TODO: CDX-533: use Custom error instead
+  private async displaySnapshotError(
+    snapshot: Snapshot,
+    reporter: SnapshotReporter
+  ) {
+    cli.log(ReportViewerStyles.error(reporter.resultCode));
+    cli.log();
```
Comment (louis-bompart) [inline packages/cli/src/commands/org/config/monitor.ts:90]: 🤨 why?

File: packages/cli/src/commands/org/config/monitor.ts [hunk 2]
```diff
@@ -63,14 +66,38 @@ export default class Monitor extends Command {
...
+    const cfg = await this.configuration.get();
+    const urlBuilder = new SnapshotUrlBuilder(cfg);
+    const snapshotUrl = urlBuilder.getSnapshotPage(snapshot);
+
+    cli.error(
+      dedent`Invalid snapshot - ${snapshot.latestReport.resultCode}.
+
+        You can also use this link to view the snapshot in the Coveo Admin Console
```
Comment (louis-bompart) [inline packages/cli/src/commands/org/config/monitor.ts:98]:
```suggestion
        You can also use this link to view the snapshot in the Coveo Admin Console:
```

**Review by louis-bompart — APPROVED** [review]
lgtm overall, some interrogation?

File: packages/cli/src/commands/org/config/monitor.ts (hunk 1 above)
Comment (y-lakhdar) [reply packages/cli/src/commands/org/config/monitor.ts:90]: Just to add some space and prevent stuff from being crowded.
I am addressing that in CDX-533

**Pull request merged**